examples/deployment/README.md

```diff
@@ -0,0 +1,731 @@
+
+## Create a secret
+
+If all of your images can be pulled from public repositories, this step is not
+required. Otherwise, you need to enable your Kubernetes cluster to pull
+images from private repositories. You must create a secret to convey
+the docker credentials to Kubernetes. In the examples,
+the secret named `coherence-example-secret` is used in the namespace `coherence-example-ns`.
+
+```bash
+kubectl -n coherence-example-ns \
+  create secret docker-registry coherence-example-secret \
+  --docker-server=$DOCKER_REPO \                              <1>
+  --docker-username=$DOCKER_USERNAME \                        <2>
+  --docker-password=$DOCKER_PASSWORD \                        <3>
+  --docker-email=$DOCKER_EMAIL                                <4>
+```
+
+* Replace `$DOCKER_REPO` with the name of the Docker repository that the images are to be pulled from.
+* Replace `$DOCKER_USERNAME` with your username for that repository.
+* Replace `$DOCKER_PASSWORD` with your password for that repository.
+* Replace `$DOCKER_EMAIL` with your email (or even a fake email).
+
+See the [Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/)
+on pull secrets for more details.
+
+## Clone the GitHub repository
+
+The examples exist in the `2.0` branch of the Coherence Operator GitHub repository - https://github.com/oracle/coherence-operator.
+
+Clone the repository and switch to the `2.0` branch:
+
+```bash
+git clone https://github.com/oracle/coherence-operator
+
+cd coherence-operator
+
+git checkout 2.0
+
+cd examples
+```
+
+In the `examples` root directory, check the pom.xm) and verify that the value of the `coherence.version` property 
+matches the version of Coherence that you are actually using. For example, if you have Coherence 12.2.1.4.0, then the value of `coherence.version` must
+be `12.2.1-4-0`.  If this value needs adjustment, use the `-Dcoherence.version=` argument for all invocations of `mvn`.
+
+Ensure you have Docker running and your Mavent and JDK11 build environemnt set and use the 
+following command to build the projects and associated Docker images:
+
+```bash
+mvn clean install -P docker
+```
+
+This will result in the following Docker image being created which contains the configuration and server-side 
+artifacts to be use by all deployments.
+
+```console
+deployment-example:2.0.0
+```
+## Install the Coherence Operator
+
+For this example we are going to install the Coherence Operator with metrics enables as well as 
+EFK (Elasticsearch, FluentD and Kiabana) configured for log capture and analysis in Kibana.
+
+If you are short on resources, then you can set `installEFK=false` and `prometheusoperator.enabled=false`.
+
+> Note: If this is the first time you have installed with Prometheus enabled, you should set `createCustomResource=true`.   
+
+> Note: The following uses the `coherence-unstable` helm repository as well as sets a specific pre-release version.
```

Comment:
You shouldn't be using the unstable repository in the docs, once it is release they should be using 2.0.0 from the release repo

Reply:
yes, understand, was just getting things to work.